Makefile

```
@@ -224,8 +224,12 @@ golint: golangci-lint
 e2e: ginkgo
 	$(MAKE) e2e-build && $(MAKE) e2e-exec && $(MAKE) e2e-destroy
 
+e2e-install-deps:
+	@$(KUBECTL) apply --force-conflicts --server-side=true -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.3.0/standard-install.yaml
```

This should auto update api gateway

```suggestion
API_GW         := none
API_GW_VERSION := v1.3.0
API_GW_LOOKUP  := kubernetes-sigs/gateway-api/
e2e-install-deps:
	@$(KUBECTL) apply --force-conflicts --server-side=true -f https://github.com/$(API_GW_LOOKUP)/releases/download/$(API_GW_VERSION)/standard-install.yaml
```